src/ui/components/SuggestedResults/index.js

```diff
@@ -0,0 +1,54 @@
+/* @flow */
+/* eslint react/no-array-index-key: 0 */
+import React from 'react';
+import escapeStringRegexp from 'escape-string-regexp';
+
+import './style.scss';
+
+type Props = {
+  query: string,
+  suggestions: Array<string>,
+  onSelect: Function,
+};
+
+const renderSuggestion = (query: string, suggestion: string, onSelect: Function) => {
```

Comment:
This is actually a simple function that returns a component indeed. I did that to highlight the query in the suggestions like it is depicted in the mockups.